start_esp/start_esp.py

```diff
@@ -1046,6 +1047,11 @@ def enforce_conflict_args(args):
             logging.error("[ESP] --server_config_generation_path must end with / when --service specifies multiple services")
             sys.exit(3)
 
+        services = args.service.split('|')
```

Comment:
Can we assign to args.services directly?   "services" variable is not used, is that right?

Reply:
good call.